notes/lecture25.tex

```diff
@@ -0,0 +1,180 @@
+\end{proposition}
+\begin{proof}
+See Proposition 5.1.1 of \cite{bertsekas2016nonlinear}.
+\end{proof}
+
+As $\domain=\R^n$, Newton's method can be applied with properly selected stepsize to ensure that all iterates lies in $S$. Concretely, an initial interior point can be obtained for some large $\epsilon_0$. Then in each iteration, we can use $x_t$ as an initialization to find $x_{t+1}$ by Newton's method. When $\epsilon_0$ is large, it is easier to find a interior point. And since $x_t$ is close to $x_{t+1}$, it is likely that $x_t$ is in the local convergence region for Newton's method. Therefore, intuitively, global convergence of Newton's method can be enabled by using barrier methods.
+
+
+\end{definition}
+\subsection{Linear Programming}
+We'll now adopt the logarithmic barrier method to solve the linear programming (LP) problem defined as follows:
+\begin{equation}
+\label{eq:LP_def}
+\text{LP}:\qquad
+\begin{aligned}[c|c|c]
+&\min_x &c^T&x\\
+&\qquad\text{s.t.} &Ax&\ge b
+\end{aligned}
+\end{equation}
+where $A\in\mathbb{R}^{m\times n}$ with $m\ge n$ and rank($A$)=n. Denote $x^*$ as the optimal point. 
+
+First we write out the augmented cost function by the logarithmic barrier method, i.e.,
+\begin{equation}
+\label{eq:LP_aug_cost}
+f_\epsilon(x) = c^T x-\epsilon \sum_{j=1}^m\ln\left(A^T_j x-b\right).
+\end{equation}
+where $A^T_j$ is the $j$-th row of $A$. Define $x^*_\epsilon=\argmin_x f_\epsilon(x)$. 
+
+\noindent\textbf{Fact}: $x^*_\epsilon$ exists and is unique for any $\epsilon>0$.
+\begin{proof}
+We can easily check that $f_\epsilon(x)$ is convex (as a sum of two convex functions). Therefore, the minimizer $x^*_\epsilon$ must exist and is unique.
+
+\textit{Remark}: For the convexity of $f_\epsilon$, we can also check the second-order derivative, which is positive definite as shown in (\ref{eq:hessian_f_eps}) later.
+\end{proof}
+
+\subsubsection{Central Path}
+The central path of the LP problem in \ref{eq:LP_def} is depicted by the set of $\left\{x^*_\epsilon|\epsilon>0\right\}$, as shown in Fig.~\ref{fig:central_path}.
+\begin{figure}[h!]
+\centering
+\begin{tikzpicture}
+\draw[gray, thick] (-1,2) -- (2,2) -- (3,-1) -- (0.5,-2) -- (-2,-1) -- cycle;
+\filldraw[black] (0,0) circle (2pt) node[anchor=west] {$x^*_\infty$};
+\filldraw[red] (0.5,-2) circle (2pt) node[anchor=north] {$x^*$};
+\draw[blue, thin] (0,0) .. controls (-0.5,-1) .. (0.5,-2);
+\draw[red, thin] (0,0) -- (-0.5,-0.5) -- (-0.1,-0.8) -- (-0.4,-1.1) -- (0.05,-1.4) -- (-0.05,-1.6) -- (0.5,-2);
+\end{tikzpicture}
+\caption{The central path\label{fig:central_path}}
+\end{figure}
+
+Our goal is to design an algorithm that will approximately follow the central path. Assume that we already get a "good" enough initial point, then at every step, we apply one step of Newton's method. To guarantee that the algorithm converges, we need to answer the following two questions:
+\begin{itemize}
+\item Under what conditions that the single-step Newton method works?
```

Comment:
Sentence incorrect. Perhaps write: "Under what conditoins does the single-step Newton method work?"